CHANGELOG.md

```diff
@@ -20,6 +20,8 @@ We refer to [GitHub issues](https://github.com/JabRef/jabref/issues) by using `#
 - The search result Window won't stay on top anymore if the main Window is focused and will be present in the taskbar
 - The user can jump from the searchbar to the maintable  with `ctrl+enter`
 - Implemented [#573 (comment)](https://github.com/JabRef/jabref/issues/573#issuecomment-232284156): Added shortcut: closing the search result window with `ctrl+w`
+- Added support for over 1000 1.0.1 CitationStyles
```

Comment:
Maybe reformulate to 1000 citation styles (using CitationStyles 1.0.1) and add link.